docker/build-tools/Dockerfile

```diff
@@ -509,7 +509,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     ruby3.2-dev
 
 # Install istio.io verification tools
-RUN gem install --no-wrappers --no-document mdl -v ${MDL_VERSION}
+RUN npm install --global markdownlint-cli2@${MARKDOWNLINT_CLI2_VERSION}
```

Comment:
This should get moved to the NodeJS build stage container and out of the Ruby one.